### PR TITLE
Update README content for org

### PR DESCRIPTION
**Pull request by Daniel-Warner-X**
This updates and revises the information of the org page `https://github.com/getappmap`

You can view the new version here: https://github.com/getappmap/.github/blob/update-readme-content-for-org/profile/README.md

NEW:

image

OLD:

image

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
+    <picture >
+      <source srcset="https://github.com/getappmap/.github/raw/main/profile/images/appmap-full-logo.svg" media="(prefers-color-scheme: light)" alt="AppMap Logo">
+      <img src="https://github.com/getappmap/.github/raw/main/profile/images/appmap-full-logo-wht.svg">
+    </picture>
 </p>
 
 --- 
 
-[AppMap](https://appmap.io) is a free and open-source runtime code analysis tool with support for [Ruby](https://github.com/getappmap/appmap-ruby), [Java](https://github.com/getappmap/appmap-java), [Python](https://github.com/getappmap/appmap-python), [Javascript](https://github.com/getappmap/appmap-agent-js)  
+[AppMap](https://appmap.io) is a versatile open-source runtime code analysis tool compatible with for [Ruby](https://github.com/getappmap/appmap-ruby), [Java](https://github.com/getappmap/appmap-java), [Python](https://github.com/getappmap/appmap-python), and [Javascript](https://github.com/getappmap/appmap-agent-js). It records traces of code execution, gathering insights into how your code functions and its behavior. These traces are then analyzed to report on significant changes in code behavior.
```
Comment (kgilpin) [inline profile/README.md]: with for

File: profile/README.md [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
-<p align="center">
-  <a href="https://marketplace.visualstudio.com/items?itemName=appland.appmap" target="_blank"><img src="https://github.com/getappmap/.github/raw/main/profile/images/VSCode.png" alt="AppMap for VS Code" style="width:300px;"></a>
-  <a href="https://plugins.jetbrains.com/plugin/16701-appmap" target="_blank"><img src="https://github.com/getappmap/.github/raw/main/profile/images/JetBrains.png" alt="AppMap for JetBrains" style="width:300px;"></a>
-</p>
+**Key Features of AppMap:**
 
-AppMap records code execution traces, collecting information about how your code works and what it does. Then it presents this information as interactive diagrams that you can search and navigate. In the diagrams, you can see exactly how functions, web services, data stores, security, I/O, and dependent services all work together when application code runs.
+- **Test Failure Analysis:** Identify the root causes of test failures.
+- **Security and Performance Flaw Detection:** Detect security vulnerabilities and performance flaws in the code.
+- **Identification of Important API Changes:** Inspect crucial alterations in the API.
```
Comment (kgilpin) [inline profile/README.md:17]: SQL

Comment (Daniel-Warner-X) [reply profile/README.md:17]: I separated the performance and security category, and got a little more specific.

File: profile/README.md [hunk 3]
```diff
@@ -1,35 +1,29 @@
...
+    <picture >
+      <source srcset="https://github.com/getappmap/.github/raw/main/profile/images/appmap-logo-blk-type.png" media="(prefers-color-scheme: light)" alt="AppMap Logo">
+      <img src="https://github.com/getappmap/.github/raw/main/profile/images/appmap-logo-wht-type.png" width="500px">
+    </picture>
 </p>
 
 --- 
 
-[AppMap](https://appmap.io) is a free and open-source runtime code analysis tool with support for [Ruby](https://github.com/getappmap/appmap-ruby), [Java](https://github.com/getappmap/appmap-java), [Python](https://github.com/getappmap/appmap-python), [Javascript](https://github.com/getappmap/appmap-agent-js)  
+[AppMap](https://appmap.io) is a versatile open-source runtime code analysis tool compatible with [Ruby](https://github.com/getappmap/appmap-ruby), [Java](https://github.com/getappmap/appmap-java), [Python](https://github.com/getappmap/appmap-python), and [Javascript](https://github.com/getappmap/appmap-agent-js). It records traces of code execution, gathering insights into how your code functions and its behavior. These traces are then analyzed to report on significant changes in code behavior.
```
Comment (kgilpin) [inline profile/README.md]: To say Node.JS is probably better.

File: profile/README.md [hunk 4]
```diff
@@ -1,35 +1,29 @@
...
-  <a href="https://plugins.jetbrains.com/plugin/16701-appmap" target="_blank"><img src="https://github.com/getappmap/.github/raw/main/profile/images/JetBrains.png" alt="AppMap for JetBrains" style="width:300px;"></a>
-</p>
+**Key Features of AppMap:**
 
-AppMap records code execution traces, collecting information about how your code works and what it does. Then it presents this information as interactive diagrams that you can search and navigate. In the diagrams, you can see exactly how functions, web services, data stores, security, I/O, and dependent services all work together when application code runs.
+- **Test Failure Analysis:** Identify the root causes of test failures.
+- **Security Flaw Detection:** Vulnerabilities like authorization logic flaws, secrets in logs, and unsafe data passed to system calls are reported.
+- **Performance Flaw Detection:** Issues like N+1 queries and slow operations can be visualized within interactive flame graphs, without the need to change application code to record spans.
```
Comment (kgilpin) [inline profile/README.md]: Elizabeth suggested “Security Flaw” and “Performance Problem”

**Review by kgilpin — APPROVED** [review]

**Pull request merged**